build.zig

```diff
@@ -23,52 +23,98 @@ pub fn build(b: *std.Build) void {
 
     const ssl = boringssl.artifact("ssl");
     const crypto = boringssl.artifact("crypto");
-    // const lshpack_c = lshpack_dep.path("lshpack.c");
-    // const lsqpack_c = lsqpack_dep.path("lsqpack.c");
 
-    const lib = b.addLibrary(.{
+    const lib = b.addStaticLibrary(.{
```

Comment:
You might use addLibrary with .linkage = .static (since addStaticLibrary is deprecated).